### PR TITLE
Minor fix on the UI.

### PR DESCRIPTION
**Pull request by LiMengyang990726**
The following minor fixes on the UI are made:
- Font size of texts in the landing page.
- Log in button at the end of landing page now goes to `/login`
- All pages nav bars are working
- `Play even harder` always direct to the `index.html` now which makes more sense to me
- Deleted unnecessary nav links and changed to what we required for our prototype.
   - `Your Profile` goes to `user-info.html`
   - `Your Page` goes to `user-page.html`
   - `See Around` goes to previously `community.html`, now renamed to `see-around.html`
   - `One Glance` goes to `statistics.html`
   - Redirect `Map` to  `Google Map`
   - `Bugs` goes to `bugs-issues.html` which is under construction for now.

### CODE REVIEW

**Comment by LiMengyang990726** [comment]
image
image

File: src/main/webapp/bugs-issues.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```
Comment (nanda-thant-sin) [inline src/main/webapp/bugs-issues.html]: What is this page used for?

File: src/main/webapp/js/navigation-loader.js [hunk 2]
```diff
@@ -86,24 +85,29 @@ function addLoginOrLogoutLinkToNavigation() {
...
         );
 
         navigationElement.appendChild(
-          createListItem(createLink('/stats.html', 'Statistics'))
+          createListItem(createLink('https://www.google.com/maps/search/google+map/', 'Map'))
         );
 
         navigationElement.appendChild(
-          createListItem(createLink('/community.html', 'Community'))
+          createListItem(createLink('/bugs-issues', 'Bugs'))
```
Comment (nanda-thant-sin) [inline src/main/webapp/js/navigation-loader.js]: Need to remove this one ,too.

File: src/main/webapp/js/navigation-loader.js [hunk 3]
```diff
@@ -86,24 +85,29 @@ function addLoginOrLogoutLinkToNavigation() {
...
         );
 
         navigationElement.appendChild(
           createListItem(createLink('/logout', 'Logout'))
         );
+
+        // Probably this is not needed in our prototype
+        // navigationElement.appendChild(
+        //   createListItem(createLink('/feed.html', 'Public Feeds'))
```
Comment (nanda-thant-sin) [inline src/main/webapp/js/navigation-loader.js]: Is it Public Feeds is that main feature where users can see everyone's messages?

**Pull request merged**